documentation/src/main/doc/modules/kafka/pages/kafka.adoc

```diff
@@ -22,6 +22,7 @@ include::inbound.adoc[]
 include::outbound.adoc[]
 include::default-configuration.adoc[]
 include::avro-configuration.adoc[]
+include::kafka-protobuf-configuration.adoc[]
```

Comment:
I personally think the `kafka-` prefix is not necessary. It's in a `kafka` directory, after all. For Avro, we have `avro-configuration.adoc`, so for Protobuf, I'd say `protobuf-configuration.adoc` should be enough.